packages/jest-environment-jsdom/package.json

```diff
@@ -15,10 +15,10 @@
     "@jest/types": "^25.1.0",
     "jest-mock": "^25.1.0",
     "jest-util": "^25.1.0",
-    "jsdom": "^15.1.1"
+    "jsdom": "^16.2.0"
```

Comment:
💡 Suggestion:

```suggestion
    "jsdom": "^16.2.1"
```
`jsdom@16.2.0` has `acorn-globals@4.3.1` as a dependency. That relies on `acron@6.1.1`, which has the following vulnerabilty: https://www.npmjs.com/advisories/1488. `jsdom@16.2.1` is almost identical, except it has the version of `acorn-globals` with the fixed version of `acorn`: https://github.com/jsdom/jsdom/compare/16.2.0...16.2.1

It _shouldn't_ have an impact on this PR except to eliminate a bunch of vulnerability warnings in dependent projects 😄.

Reply:
Thanks @mindblight, I will check and update it!

Reply:
Hi @lamhieu-vk !  I have created [a new PR](https://github.com/lamhieu-vk/jest/pull/1) for you with the suggested change, can you merge it? We need this update to be able to test our components in our real Angular project. We have components using *quill* library which depends on these DOM functions. Thanks so much!

Reply:
thank you, I merged it